doc/irmin-pack/design/lower_layer.md

```diff
@@ -75,39 +75,39 @@ Since it contains a contiguous commit history, there are no gaps of commits from
 
```

Comment:
Li 74  :: Could this be: 
"Since it contains a contiguous commit history, there are no commit gaps between its first and last commit" 
?